packages/server/graphql/intranetSchema/mutations/hardDeleteUser.ts

```diff
@@ -0,0 +1,162 @@
+import {GraphQLID, GraphQLNonNull, GraphQLString} from 'graphql'
+import {GQLContext} from '../../graphql'
+import {requireSU} from '../../../utils/authorization'
+import getRethink from '../../../database/rethinkDriver'
+import getPg from '../../../postgres/getPg'
+
+const hardDeleteUser = {
+  type: GraphQLNonNull(GraphQLString),
+  description: 'hard deletes a user and all its associated objects',
+  args: {
+    userId: {
+      type: GraphQLNonNull(GraphQLID)
+    },
```

Comment:
decided to only go for `userId` arg instead of allowing `email` option as well. with rethink, i feel like it may still be possible to have duplicate emails. even though it's not as "user friendly", it seems safer to only delete by id. anyone calling this mutation should be able to specify the id.